coral/README.md

```diff
@@ -90,32 +92,17 @@ The script `lint` runs a prettier check and eslint after. It does not mutate you
 ## Styling
 
 We use the component library of Aiven's design system:
+
 - 📃 [documentation](https://aiven-ds.netlify.app/)
 - the repository is open source, but `private` at the moment
 
 As a rule, please don't use css classes from the design system. All styles should be created by using the existing components and their properties.
 
-__🔄 Work in progress related to styles__
+**🔄 Work in progress related to styles**
+
 - We plan to add css variables based on the design system's tokens.
 - We plan to have a custom theme for Klaw. This will be used instead of the Aiven theme.
 
 ## Documentation
 
-We've a more detailed document about our thinking about [Docmentation](docs/documentation.md).
-
-### Tip
-You can use `alex` ( 📃 [documentation]( https://alexjs.com/) | 🐙 [repository](https://github.com/get-alex/alex)) as an **optional** linting tool. It checks text documents and highlights language that potentially could be insensitive or inconsiderate.
-
-- run it with `pnpm --silent dlx alex` to lint text files.
-
-
-## More detailed documentation
-
-We provide more documentation on:
-
-- Our commitment to [Accessibility](docs/accessibility.md)
-- Detailed overview of the [Directory Structure](docs/directory-structure.md)
-- Our thinking about [Docmentation](docs/documentation.md)
-- More information about [Frontend Testing](docs/frontend-testing.md)
-- [Development with remote API](docs/development-with-remote-api.md)
-
+[More detailed documentation](docs) about `coral` can be found in the `docs` folder.
```

Comment:
Maybe we can add the `coral/README.md` as a link here? That way, the user will be right in another md file with a TOC and can navigate from there.

Reply:
Yes, I think it makes more sense than linking to the root and having to scroll down to see the `README` indeed.

Reply:
Yes, that's what I meant! :D